opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp

```diff
@@ -128,170 +128,183 @@ class SimulatorFullyImplicitBlackoilEbos
     /// \param[in,out] state       state of reservoir: pressure, fluxes
     /// \return                    simulation report, with timing data
     SimulatorReport run(SimulatorTimer& timer)
+    {
+        runInit(timer);
+        // Main simulation loop.
+        while (!timer.done()) {
+            runStep(timer);
+        }
+        return runLastStep();
```

Comment:
The name `runLastStep()` is misleading because it does not run a step. Perhaps `finalize()` is better?